### PR TITLE
 Hyper-V process detection on Windows

### PR DESCRIPTION
**Pull request by kmazurek**
On Windows, if Hyper-V is enabled, WMI queries report lack of
virtualization support. To correctly detect virtualization capabilities,
we need to first check if Hyper-V process is running on the host.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -78,7 +78,7 @@ Pillow==4.2.1
 pluggy==0.6.0
 portalocker==1.2.1
 priority==1.3.0
-psutil==5.2.2
+psutil==5.4.8
```
Comment (mfranciszkiewicz) [inline requirements.txt]: I'd suggest not to update dependencies now unless there is a strong reason to do so

**Comment by Wiezzel** [comment]
You can check much easier if Hyper-V is enabled. It is actually already done in `HyperVHypervisor.is_available()` method: https://github.com/golemfactory/golem/blob/6ec4e13fba79b011864d1b453afc78716164c31c/golem/docker/hypervisor/hyperv.py#L194-L202
So just modify `_check_vt_windows()` or `get-virtualization-state.ps1` to include this command.

**Comment by kmazurek** [comment]
@Wiezzel Thanks, didn't notice that. I saw another solution using `Get-WindowsOptionalFeature`, but that one requires running as admin. Let me update and test this.

File: requirements.txt (hunk 1 above)
Comment (kmazurek) [reply requirements.txt]: Got it, turns out it won't be required most likely (see comment below).

**Review by Wiezzel — APPROVED** [review]
👍

**Review by mfranciszkiewicz — APPROVED** [review]

**Pull request merged**